.github/scripts/transform_dbt_test_results.py

```diff
@@ -4,64 +4,10 @@
 # other teams for review and correction, along with metadata parquet files
 # that can be uploaded to S3 for long-term result tracking.
 #
-# This script assumes that it is being run in sequence after a call to
-# `dbt test --store-failures`, since it depends on two files created by
-# that operation (target/run_results.json and target/manifest.json).
-# It also requires Python dependencies be installed from
-# requirements.transform_dbt_test_results.txt.
```

Comment:
All of this text was preserved, just moved into the CLI documentation for more idiomatic access.